perf.groovy

```diff
@@ -341,15 +408,18 @@ def static getOSGroup(def os) {
                 }
             }
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("bash ./tests/scripts/perf-prep.sh --throughput")
```

Comment:
I think that my change would probably to have another job, on Windows, that does the nuget install step that we do on Windows now, to pull down the TP benchmarks package, archive the binaries, and then have the test job grab the binaries from that leg. It would be in parallel to the build.  OR someone can teach me how the build system works so we could have it pull down the package somehow.